packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -292,6 +292,10 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Sets the active camera while recording.
+  ///
+  /// On Android, calling this method will normally cancel any active recording.
```

Comment:
Nit (for all of the places where this is pasted): I think "normally" is a bit vague. Wording it something like this may be more clear:

```txt
On Android, you must start the recording with [startVideoCapturing] with
`enablePersistentRecording` set to `true` to avoid canceling any active recordings
when this method is called.
```